modules/nf-core/bowtie2/align/main.nf

```diff
@@ -2,7 +2,7 @@ process BOWTIE2_ALIGN {
     tag "$meta.id"
     label 'process_high'
 
-    conda "${moduleDir}/environment.yml"
+    conda "${moduleDir}/conda-lock.yml"
```

Comment:
```suggestion
    conda "${moduleDir}/conda-lock-linux-64.yml"
```

Reply:
(I'm obviously not proposing we actually change modules for platform specificity, just illustrating).

Reply:
I think tracking them at this point might keep us sane though 😅

Reply:
Because the lock files will be platform specific they can't go directly on the modules. I think we'd need platform-specific configs at the workflow level that referenced all the lockfiles.